### PR TITLE
Measure PQ192-HH

### PR DESCRIPTION
**Pull request by abelzer212**
## Purpose

Bring in new measure PQ192-HH

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-9906

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Review by jdinh8124 — APPROVED** [review]
Looks good to me

File: services/ui-src/src/measures/2021/PQI92HH/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          {showOptionalMeasureStrat && (
+            <CMQ.OptionalMeasureStrat
+              categories={PMD.categories}
+              qualifiers={PMD.qualifiers}
+              rateMultiplicationValue={100000}
+              customMask={positiveNumbersWithMaxDecimalPlaces(1)}
+              performanceMeasureArray={performanceMeasureArray}
+              adultMeasure
+              allowNumeratorGreaterThanDenominator
+            />
```
Comment (jphechter) [inline services/ui-src/src/measures/2021/PQI92HH/index.tsx:75]: Missing `calcTotal`

**Review by jphechter — CHANGES_REQUESTED** [review]
Needs one change, otherwise LGTM 👍

File: services/ui-src/src/measures/2021/PQI92HH/index.tsx (hunk 1 above)
Comment (abelzer212) [reply services/ui-src/src/measures/2021/PQI92HH/index.tsx]: Just added that back in

**Review by jphechter — DISMISSED** [review]
LGTM 👍

**Pull request merged**